pytorch_fairseq_translation.md

```diff
@@ -10,6 +10,8 @@ author: Facebook AI (fairseq Team)
 tags: [nlp]
 github-link: https://github.com/pytorch/fairseq/
 github-id: pytorch/fairseq
+featured_image_1: no-image
+eatured_image_2: no-image
```

Comment:
this has to be featured_image_2

Reply:
Thanks Soumith, apologies for the missing letter (Copy and paste fault) I'm closing this PR since I haven't been able to merge it due to my limited permissions. I've created the change in another PR #129
Can you please help me get that change merged and approved? Thanks!